cpu/stm32/periph/backup_ram.c

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2021 Otto-von-Guericke Universität Magdeburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     cpu_stm32
+ * @{
+ *
+ * @file
+ * @brief       Implementation of backup SRAM capabilities for stm32 MCUs
+ *
+ * @author      Fabian Hüßler <fabian.huessler@ovgu.de>
+ * @}
+ */
+
+#include "periph_cpu.h"
+#include "stmclk.h"
+
+#ifndef BKPRAM_CONFIG
+/**
+ * @brief   Configuration to retain backup RAM content during standby mode
+ */
+#if defined(CPU_FAM_STM32F7)
```

Comment:
Would this also work?
```suggestion
#if defined(PWR_CSR1_BRE)
```

makes porting to new families less effort

Reply:
Yes so far only `F7` seems to have `CSR1`.